.circleci/config.yml

```diff
@@ -1491,6 +1505,7 @@ jobs:
       TERM: xterm
       MAKEFLAGS: -j 2
       LC_ALL: C
+    parallelism: 2 # For prepare_bytecode_report
```

Comment:
Why does this only parallelize `prepare_bytecode_report`, or rather, how does it know to only parallelize that particular step? Is it linked somehow?